thebeat/stats.py

```diff
@@ -609,13 +610,25 @@ def fft_values(
     The number of cycles per unit can be interpreted as the beat frequency. For instance, 2 cycles for a unit size of
     1000 ms means a beat frequency of 2 Hz.
 
+    Note
+    ----
+    If a sequence ends with an event, the last event is not included in the calculation of the Fourier transform.
+    This is because the Fourier transform assumes that the sequence can be repeated indefinitely.
+
```

Comment:
You could also phrase this in a way that it's about the intervals, not the events. Cause that's basically what it is: A potential event at the end of the sequence will be directly followed by the first event anyway.

Reply:
Oh, but more importantly! What happens for a sequence where the first event isn't at time 0? Is it padded with 0s before, or does it just start at the first events? How does this interact with this last event case?
Can we make this explicit in the docs and add a short test?

Reply:
The latter is taken into account in the ``sequence_to_binary`` function

Reply:
Fixed by later commits, great!